lib/functions/artifacts/artifact-armbian-bsp-cli.sh

```diff
@@ -56,6 +56,7 @@ function artifact_armbian-bsp-cli_prepare_version() {
 		"BOARDFAMILY: ${BOARDFAMILY}"                                 # /etc/armbian-release
 		"LINUXFAMILY: ${LINUXFAMILY}"                                 # /etc/armbian-release
 		"IMAGE_TYPE: ${IMAGE_TYPE}"                                   # /etc/armbian-release
+		"NETWORKING_TYPE: ${NETWORKING_TYPE}"                         # /etc/armbian-release
```

Comment:
This becomes inconsistent, unless also added to the hash (in the artifact).

Reply:
And then since the bsp-cli pkg starts varying depending on the networking extensions enabled, EXTRA_BSP_NAME also becomes mandatory, otherwise, packages will be inconsistent/override each other when published to apt repo.